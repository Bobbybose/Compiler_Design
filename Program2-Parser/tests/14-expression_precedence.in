a = b || c && d == 2 > 5-4*2;
