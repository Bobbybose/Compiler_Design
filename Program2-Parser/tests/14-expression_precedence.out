statementList: 1,1
|   
+-- statementList (EMPTY STRING): 1,1
|   
+-- statement: 1,1
|   |   
|   +-- expression: 1,1
|   |   |   
|   |   +-- expression: 1,1
|   |   |   |   
|   |   |   +-- a: 1,1
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- =: 1,3
|   |   |   
|   |   +-- expression: 1,5
|   |   |   |   
|   |   |   +-- expression: 1,5
|   |   |   |   |   
|   |   |   |   +-- b: 1,5
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   +-- ||: 1,7
|   |   |   |   
|   |   |   +-- expression: 1,10
|   |   |   |   |   
|   |   |   |   +-- expression: 1,10
|   |   |   |   |   |   
|   |   |   |   |   +-- c: 1,10
|   |   |   |   |   |   
|   |   |   |   |   *   
|   |   |   |   |   
|   |   |   |   +-- &&: 1,12
|   |   |   |   |   
|   |   |   |   +-- expression: 1,15
|   |   |   |   |   |   
|   |   |   |   |   +-- expression: 1,15
|   |   |   |   |   |   |   
|   |   |   |   |   |   +-- d: 1,15
|   |   |   |   |   |   |   
|   |   |   |   |   |   *   
|   |   |   |   |   |   
|   |   |   |   |   +-- ==: 1,17
|   |   |   |   |   |   
|   |   |   |   |   +-- expression: 1,20
|   |   |   |   |   |   |   
|   |   |   |   |   |   +-- expression: 1,20
|   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   +-- 2: 1,20
|   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   *   
|   |   |   |   |   |   |   
|   |   |   |   |   |   +-- >: 1,22
|   |   |   |   |   |   |   
|   |   |   |   |   |   +-- expression: 1,24
|   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   +-- expression: 1,24
|   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   +-- 5: 1,24
|   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   *   
|   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   +-- -: 1,25
|   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   +-- expression: 1,26
|   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   +-- expression: 1,26
|   |   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   |   +-- 4: 1,26
|   |   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   |   *   
|   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   +-- *: 1,27
|   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   +-- expression: 1,28
|   |   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   |   +-- 2: 1,28
|   |   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   |   *   
|   |   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   |   *   
|   |   |   |   |   |   |   |   
|   |   |   |   |   |   |   *   
|   |   |   |   |   |   |   
|   |   |   |   |   |   *   
|   |   |   |   |   |   
|   |   |   |   |   *   
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   *   
|   |   
|   +-- ;: 1,29
|   |   
|   *   
|   
*   
